### PR TITLE
Show debug info with a `dry.` parameter

### PR DESCRIPTION
**Pull request by khalidharun**

### CODE REVIEW

File: R/cache.R [hunk 1]
```diff
@@ -370,9 +391,9 @@ execute <- function(fcn_call) {
       retry = 3,
       stop = TRUE
     )
-    uncached_data <- batched_fn(uncached_keys)
+    uncached_data <- batched_fn(uncached_keys, fcn_call$force)
```
Comment (robertzk) [inline R/cache.R:401]: Where are `uncached_keys` coming from now? This will break

**Comment by robertzk** [comment]
This will break, you have to address a comment. Can you add a test?

**Comment by khalidharun** [comment]
yeah, there's lots of issues with this.  I'll take a closer look at it tomorrow.

File: R/cache.R (hunk 1 above)
Comment (khalidharun) [inline R/cache.R:401]: `uncached_key` are retrieved correctly now just above.

**Comment by khalidharun** [comment]
PR replaced by #106. Closing.

**Pull request closed without merging**